micrometer/src/main/java/feign/micrometer/FeignObservationDocumentation.java

```diff
@@ -74,6 +75,12 @@ public String asString() {
       public String asString() {
         return "net.peer.port";
       }
+    },
+    CLIENT {
+      @Override
+      public String asString() {
+        return "client";
```

Comment:
Potentially would be better named  `clientName` to match the spring RestTemplate, WebClient etc

Reply:
Makes sense

Reply:
Updated tag to `clientName`